Formula/jacoco-cli.rb

```diff
@@ -0,0 +1,18 @@
+class JacocoCli < Formula
+  desc "JaCoCo Java Code Coverage Library Command-line"
+  homepage "https://github.com/jacoco/jacoco"
+  url "https://search.maven.org/remotecontent?filepath=org/jacoco/org.jacoco.cli/0.8.7/org.jacoco.cli-0.8.7-nodeps.jar"
+  sha256 "11b549a9ef14d8454534f914ca1051fb9bcacab7f501e9f1c018eacfc5e77e8d"
+  license "EPL-2.0"
+
+  depends_on "openjdk@11"
+
+  def install
+    libexec.install "org.jacoco.cli-0.8.7-nodeps.jar"
+    bin.write_jar_script libexec/"org.jacoco.cli-0.8.7-nodeps.jar", "jacoco-cli"
+  end
+
+  test do
+    assert_match "0.8.7.202105040129", shell_output("#{bin}/jacoco-cli version")
```

Comment:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/jacoco-cli version")
```

Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Thanks for the suggestion. I noticed the requirement for testing the functionality rather than just basic outputs of the installed cask. 
Will update this test case.